src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  filterType: FilterType
+  setFilterType: (status: FilterType) => void
+  completedTodos: number
+  todosLength: number
+  onRemove: () => Promise<void>
+};
+
+export const Footer: React.FC<Props> = React.memo(({
+  filterType,
+  setFilterType,
+  completedTodos,
+  onRemove,
+  todosLength,
+}) => {
+  const uncompletedCount = todosLength - completedTodos;
+
+  const filterTypeList = useMemo(() => (
+    Object.values(FilterType)
+  ), []);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${uncompletedCount} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {filterTypeList.map(status => (
+          <a
+            data-cy="FilterLinkAll"
+            href="#/"
```

Comment:
also you may save this values to object and read it by key

```
const filterHrefByType = {
  [FilterType.All]: '#/',
  // other values
};

/* code */

href={filterHrefByType[status]}
```

Reply:
But still it's not critical, but you should have proper attribute values for this links